### ISSUE TITLE
Normalize task and resource naming before using hardened images

### ISSUE THREAD

**Issue by jameshochadel** [cloud-gov/product#2672]
Before we start migrating all our pipelines to use the new hardened images, we should rename the git repositories and Concourse pipelines for those images to follow a standard convention. 

* Each repository + pipeline + image triple should have the same name. If the repository is named `foo`, the pipeline must be named `foo`, and the image it produces must be pushed to the `foo` repository — i.e. be named `foo`.
* Git repos for Concourse resources must end with `-resource`.
* Git repos for Concourse tasks must end with `-task`. 
* Repository names should not be prefixed, e.g. with `cg-`. The `cg-` prefix is an artifact from the days when cloud.gov repositories were in the 18F organization. Now that they are in the `cloud-gov` organization, the prefix is no longer needed.

--- 

## Security considerations

None. Naming change only.

## Implementation sketch

At least two repositories must be renamed:

* `s3-resource-simple` -> `s3-simple-resource`
* [cg-deploy-concourse-docker-image](https://github.com/cloud-gov/cg-deploy-concourse-docker-image) must be simplified somehow. `generic-task` or something simple might do the trick since it's a catch-all task with lots of tools.
```[tasklist]
### Tasks
- [x] Rename s3-resource-simple -> s3-simple-resource
- [x] Rename cg-deploy-concourse-docker-image must be simplified somehow. generic-task or something simple might do the trick since it's a catch-all task with lots of tools.
- [x] Check the rest of the images to check if anything else needs to be renamed
```

**Comment by jameshochadel** [comment]
The only other one in the spreadsheet that needs to be renamed is `oracle-client`. I'll make a note in https://github.com/cloud-gov/product/issues/2612 to rename it and close this issue.
